Modify README.md file with instructions (#89)
fireblazrs/library-management-system: README.md

# Library Management System

> This is a Spring Boot application that implements books, authors, users, tasks etc. that are gathered in a MySQL database. Each role has a different degree of access to the application.
>  

Reviewer: ```suggestion
This is a Library Management System application that is built with Spring Boot which has a MySQL database and uses RabbitMQ. 

The application allows the user to create/save, update, delete and view books, book properties (genre, format), authors,  users and rooms. It automatically generates tasks which can be queried for status updates when an attempt to borrow a book is made. Admin and user roles have different levels of access in the application.
 
```

> ### How do I start the application?

Reviewer: Use statements rather than questions for headings and sub-headings. The latter is less formal.
Here is an example of README file from Baeldung https://github.com/eugenp/tutorials/tree/master/spring-web-modules/spring-thymeleaf.

```
docker-compose up --build
```
###What features are implemented?
- Database relations
- Dockerfile/Docker compose
- Create, Read, Update and Delete operations
- Customized exceptions
- Spring Boot Security 
- RabbitMQ and JMS
- Integration Testing

> - Thymeleaf 

Reviewer: ```suggestion
```

Thymeleaf will not be included

Additional features: 
MySQL database 

###How do I deploy it ?
###Option 1

> Clone the repository:
> 
> - Navigate to the folder where you want the application to be saved.
> 
> - Use your console to run the following:
> 
>    git clone https://github.com/fireblazrs/library-management-system
> 
> ###Or
> 

Reviewer: ```suggestion

```

It is preferable if a user downloads a the packaged application (as above) or an image of our application (as below). Since they are accessing our project repository on Github, it is reasonable to assume they know how to use `git clone`.

> Navigate to the latest release: https://github.com/fireblazrs/library-management-system/pkgs/container/library-management-system
> 
> Save the compressed file to the location where you want to
> store the application after unzipping it.

Reviewer: ```suggestion
Download the latest release [here](https://github.com/fireblazrs/library-management-system/pkgs/container/library-management-system)

```
`[](url)`  denotes a link in `md` format.

####To run the application:
1. Install Docker Desktop: https://www.docker.com/products/docker-desktop/

> 2. Navigate to the application's folder and run the following 
> command from your console: 
> docker-compose up

Reviewer: ```suggestion
```
Repetition

###Option 2 

> Download Docker Image:
> 
> 1. Install Docker Desktop:
>    https://www.docker.com/products/docker-desktop/
> 2. Create a docker-compose.yml file as follows:
>    
>          version: "2.1"
>          services:
>            mysql:
>              image: mysql:8.0.28
>              restart: always
>              ports:
>                - "3307:3306"
>              environment:
>                MYSQL_ROOT_HOST: "%"
>                MYSQL_ROOT_PASSWORD: root
>                MYSQL_USER: user
>                MYSQL_PASSWORD: password
>                MYSQL_DATABASE: test
>            server:
>              build: .
>              depends_on:
>                - mysql
>              ports:
>                - "8080:8080"
>              environment:
>                DB_HOST: mysql
>                DB_USER: user
>                DB_PASSWORD: password
>            rabbit_queue:
>              image: rabbitmq:3-management
>              ports:
>                - "5672:5672"
>                - "15672:15672"

Reviewer: ```suggestion
Download Docker Image:

    docker pull ghcr.io/fireblazrs/library-management-system:latest
```
It is preferable if our users pull an image of our packaged application (similar to the README files of real projects).

> 
> ####To run the application:
> 1. Navigate to the folder of the docker-compose.yml file and run the following
>    command from your console:
>    docker-compose up

Reviewer: ```suggestion

```
Repetition of line 10.

> ###How do i reach the application? 

Reviewer: Convert question to statement
